.github/workflows/ci.yml

```diff
@@ -122,16 +127,16 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.8"]
+        python-version: ["3.8", "3.12"]
         db-backend: ["postgresql"]
         nautobot-version: ["2.0.6", "2.1", "2.2"]
         include:
           - python-version: "3.11"
             db-backend: "postgresql"
             nautobot-version: "2.0.6"
-          # - python-version: "3.11"
-          #   db-backend: "mysql"
-          #   nautobot-version: "stable"
+          - python-version: "3.12"
+            db-backend: "mysql"
+            nautobot-version: "stable"
```

Comment:
Mysql support was disabled in https://github.com/nautobot/nautobot-app-netbox-importer/pull/110 but I don't see any justification documented. I think we **should** keep it disabled or at least change it to `2.2` instead of `stable` as long as Nautobot v2.3 support hasn't been officially added to this app.